src/librustdoc/html/render/print_item.rs

```diff
@@ -263,11 +263,30 @@ fn item_module(w: &mut Buffer, cx: &Context<'_>, item: &clean::Item, items: &[cl
             }
 
             clean::ImportItem(ref import) => {
+                let mut stab = None;
+                let mut stab_tags = None;
+
+                if let Some(did) = import.source.did {
+                    // Just need an item with the correct def_id
+                    let mut import_item = myitem.clone();
+                    import_item.def_id = did;
+                    stab = import_item.stability_class(cx.tcx());
+                    stab_tags = Some(extra_info_tags(&import_item, item, cx.tcx()));
+                }
+
+                let add = if stab.is_some() { " " } else { "" };
+
                 write!(
                     w,
-                    "<tr><td><code>{}{}</code></td></tr>",
-                    myitem.visibility.print_with_space(cx.tcx(), myitem.def_id, cx.cache()),
-                    import.print(cx.cache(), cx.tcx()),
+                    "<tr class=\"{stab}{add}module-item\">\
```

Comment:
Are you sure `module-item` is correct here? These aren't modules, they're imports.

Reply:
Introduced an `import-item` class.